snapcraft/plugins/_python/_python_finder.py

```diff
@@ -0,0 +1,44 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import glob
+
+
+def get_python_command(python_version, stage_dir, install_dir):
```

Comment:
can you please add docstrings to these two functions?

Reply:
Yes, definitely. We should try to start making that a habit, at least on public functions. Agreed?

Reply:
I'm +1. Please point it out when I forget it.

Reply:
You got it.

Reply:
Done.